src/test/java/seedu/address/logic/commands/CommandResultTest.java

```diff
@@ -4,10 +4,66 @@
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+
 public class CommandResultTest {
+
+    private static Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_eventsCommand_commandResultIsEvent() throws CommandException {
```

Comment:
If I understand this right, this is to check if a result type is `event` based right? If so, naming to something that is more aligned with that, like maybe `forEvent` may be clearer.

Reply:
yup makes sense, ok!